### ISSUE TITLE
init fails with chmod type error

### ISSUE THREAD

**Issue by s9w** [EMPRI-DEVOPS/git-privacy#37]
Python 3.9.7, win10

```
Traceback (most recent call last):
  File "C:\Anaconda3\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Anaconda3\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Anaconda3\Scripts\git-privacy.exe\__main__.py", line 7, in <module>
  File "C:\Anaconda3\lib\site-packages\click\core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "C:\Anaconda3\lib\site-packages\click\core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "C:\Anaconda3\lib\site-packages\click\core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "C:\Anaconda3\lib\site-packages\click\core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "C:\Anaconda3\lib\site-packages\click\core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "C:\Anaconda3\lib\site-packages\click\decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "C:\Anaconda3\lib\site-packages\gitprivacy\gitprivacy.py", line 131, in do_init
    copy_hook(git_dir, "post-commit")
  File "C:\Anaconda3\lib\site-packages\gitprivacy\gitprivacy.py", line 185, in copy_hook
    os.chmod(dst.fileno(), stat.S_IRWXU | stat.S_IRGRP | stat.S_IXGRP |
TypeError: chmod: path should be string, bytes or os.PathLike, not int
```

**Comment by cburkert** [comment]
Thanks for reporting. File descriptor arguments are not supported win Win, hence the issue. I'm working on it.

**Comment by cburkert** [comment]
@s9w It should be fixed in version 2.2. If not, please re-open.

**Comment by fapdash** [comment]
Verified fixed.
